ister_test.py

```diff
@@ -987,6 +1088,58 @@ def mock_listdir(directory):
     commands_compare_helper(commands)
 
 
+@cryptsetup_wrapper
+@run_command_wrapper
+def setup_mounts_encryption_good():
+    """Setup mount points for install"""
+    backup_mkdtemp = tempfile.mkdtemp
+    backup_listdir = os.listdir
+
+    def mock_mkdtemp(*_, **kwargs):
+        """mock_mkdtemp wrapper"""
+        if not kwargs.get("prefix"):
+            raise Exception("Missing prefix argument to mkdtemp")
+        COMMAND_RESULTS.append(kwargs["prefix"])
+        return "/tmp"
+
+    def mock_listdir(_):
+        return ['sda', 'sda1']
+
+    tempfile.mkdtemp = mock_mkdtemp
+    os.listdir = mock_listdir
+    template = {"PartitionMountPoints": [{"mount": "/", "disk": "sda",
+                                          "partition": 1,
+                                          "encryption" : {
+                                            "name" : "mapper_name"}},
+                                         {"mount": "/boot", "disk": "sda",
+                                          "partition": 2}],
+                "FilesystemTypes": [{"disk": "sda", "partition": 1,
+                                     "type": "vfat",
+                                          "encryption" : {
+                                            "name" : "mapper_name",
+                                            "passphrase" : "123"}},
+                                    {"disk": "sda", "partition": 2,
+                                     "type": "ext4"}],
+                "Version": 10}
+    commands = ["ister-10-",
```

Comment:
Hrm I'm not sure where this 'ister-10-' is coming from, mind explaining this one for me?

Reply:
me neither I took it from the other setup_mounts_good

Reply:
I just add additional output expected with the changes that I provide the template

Reply:
Okay, I looked it up, just something setup_mounts is doing for the tempdir (ister-$version-)